src/beam/views.py

```diff
@@ -355,6 +355,63 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class InlineActionMixin(InlinesMixin):
+    def get_action_qs(self, inline, action):
```

Comment:
Why is there a parameter action?

Reply:
And if there is a reason (flexibility in qs generation), than maybe we want such a parameter for the vies.ListAction(s)Mixin too?